src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -405,9 +405,12 @@
     type: command
     short-summary: Creates a new task which can be triggered by git commits or base image updates.
     examples:
-        - name: Create a task which updates on git commits and base image updates.
+        - name: Create a docker build task
           text: >
-            az acr task create -t helloworld:{{.Run.ID}} -n helloworld -r myRegistry -c https://github.com/Azure-Samples/acr-build-helloworld-node --git-access-token 0000000000000000000000000000000000000000
+            az acr task create -t helloworld:{{.Run.ID}} -n helloworld -r myRegistry -c https://github.com/Azure-Samples/acr-build-helloworld-node.git --git-access-token 0000000000000000000000000000000000000000 -f Dockerfile
+        - name: Create a task using task and values file.
+          text: >
+            az acr task create -n helloworld -r myRegistry -c https://github.com/AzureCR/acr-task.git --git-access-token 0000000000000000000000000000000000000000 -f ./examples/templating/acb.yaml --values ./examples/templating/prod-values.yaml
```

Comment:
Probably should PR this to the Azure-Samples repo, since I think this will be deprecated in the future.

Reply:
yes .. please don't use AzureCR in any sample or help text.